packages/Webkul/Shop/src/Resources/views/components/layouts/header/mobile/index.blade.php

```diff
@@ -34,8 +34,60 @@
                 </x-slot:header>
 
                 <x-slot:content>
+                    {{-- Localization & Currency Section --}}
+                    <div class="w-full flex gap-x-[20px] justify-between items-center mb-[15px] ">
+
+                        <x-shop::dropdown position="bottom-left">
+                            <!-- Dropdown Toggler -->
+                            <x-slot:toggle>
+                                <div class="w-full flex gap-[10px] justify-between items-center cursor-pointer">
+                                    <span>
+                                        {{ core()->getCurrentCurrency()->symbol . ' ' . core()->getCurrentCurrencyCode() }}
+                                    </span>
+
+                                    <span class="text-[24px] icon-arrow-down"></span>
+                                </div>
+                            </x-slot:toggle>
+
+                            <!-- Dropdown Content -->
+                            <x-slot:content class="!p-[0px]">
+                                <v-currency-switcher></v-currency-switcher>
+                            </x-slot:content>
+                        </x-shop::dropdown>
+
+                        <x-shop::dropdown position="bottom-right">
+                            <x-slot:toggle>
+                                {{-- Dropdown Toggler --}}
+                                <div class="w-full flex gap-[10px] justify-between items-center cursor-pointer">
+                                    <img
+                                        src="{{ ! empty(core()->getCurrentLocale()->logo_url)
+                                                ? core()->getCurrentLocale()->logo_url
+                                                : bagisto_asset('images/default-language.svg')
+                                            }}"
+                                        class="h-full"
+                                        alt="Default locale"
+                                        width="24"
+                                        height="16"
+                                    />
+
+                                    <span>
+                                        {{ core()->getCurrentChannel()->locales()->orderBy('name')->where('code', app()->getLocale())->value('name') }}
+                                    </span>
+
+                                    <span class="text-[24px] icon-arrow-down"></span>
```

Comment:
Same here (Prioritise the custom class.)

Reply:
@amit-webkul Its Also Done